src/communication/http/httpClient.ts

```diff
@@ -41,7 +41,14 @@ export abstract class HttpClient {
 
     const delayFunc = axiosRetryConfig.retryDelay ?? ((): number => 0);
     axiosRetryConfig.retryDelay = (retryCount: number, error: AxiosError): number => {
-      this.logger.error(`error from ${this.targetService}. retries: ${retryCount}. error: ${error.message}`);
+      this.logger.error(
```

Comment:
insert the message as attribute of the log object => msg: "the error str" its more readable and anyway the pino converse the string argument to this.